users/ignoring.rst

```diff
@@ -20,6 +20,18 @@ If some files should not be synchronized to other nodes, a file called
 ``#include`` files that *are* synchronized between nodes. All patterns are
 relative to the repository root.
 
+.. warning::
+
+  Please note, ignored files do NOT imply garbage files such as *Thumbs.db*
```

Comment:
I think this is a bit strong. I think something less urgent like a `.. note::` with something like "Note that ignored files can block the removal of an otherwise empty directory. See below for the `(?d)` prefix to allow deletion of ignored files." or so would be cleaner